.github/workflows/validate-theme-entry.yml

```diff
@@ -0,0 +1,264 @@
+name: Validate Theme Entry
+
+on:
+  pull_request_target:
+    branches:
+      - master
+    paths:
+      - community-css-themes.json
+
+jobs:
+  theme-validation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: "refs/pull/${{ github.event.number }}/merge"
+      - uses: actions/setup-node@v2
+      - run: npm install probe-image-size
+      - uses: actions/github-script@v6
+        with:
+          script: |
+            const probe = require('probe-image-size');
+            const fs = require('fs');
+
+            if (context.payload.pull_request.additions <= context.payload.pull_request.deletions) {
+                // Don't run any validation checks if the user is just modifying existing theme config
+                return;
+            }
+
+            const errors = [];
+            const addError = (error) => {
+                errors.push(`:x: ${error}`);
+                console.log('Found issue: ' + error);
+            };
+
+            const warnings = [];
+            const addWarning = (warning) => {
+                warnings.push(`:warning: ${warning}`);
+                console.log('Found issue: ' + warning);
+            }
+
+            // core validation logic
+            await (async () => {
+                if (context.payload.pull_request.changed_files > 1) {
+                    addError('You modified files other than `community-css-themes.json`.');
+                }
+
+                const author = context.payload.pull_request.user.login;
+                const themes = JSON.parse(fs.readFileSync('community-css-themes.json', 'utf8'));
+                const theme = themes[themes.length - 1];
+
+                let prIssue = false;
```

Comment:
Can we call this `hasPullRequestError` instead? I saw "issue" at first and assumed it was related to Github Issues.